addons/grenades/stringtable.xml

```diff
@@ -530,11 +530,11 @@
         </Key>
         <Key ID="STR_ACE_Grenades_Convert_Fuse">
             <English>Convert to short fuse</English>
-            <French>Mèche lente (7 secondes)</French>
+            <French>Retardateur (7 secondes)</French>
```

Comment:
`(7 secondes)` - We should either remove from french or add to english so they are consistent

Reply:
What do you think is the best solution ?
From my point of view, I thought that indicating the duration could be interesting, because in French, it is complicated to translate this text correctly, in the space available.

Reply:
@Cyruz143

Reply:
I was trying to keep shortish since it's coming up in the inventory UI and I'm not sure how it will scale on big UI scales with lower resolutions etc